Bringing over small edits from the release that were supposed to go b… (#680)
pyansys/pyfluent: doc/source/api/core/index.rst

Core
====

> This section describes the core pythonic interfaces for Fluent. 

Reviewer: ```suggestion
This section describes the core Pythonic interfaces for Fluent. 
```

Reviewer: @smf2001 @dnwillia-work I think we still need to address terminology. I believe there is one API for Fluent-Core. It consists of multiple resources (methods, functions). If this is accurate, the above sentence should be something like "This section describes the Fluent API (application programming interface).

Reviewer 2: Yes, the ansys-fluent-core package provides what we call the 'core' APIs.

I would probably write "This section describes the Fluent Python API (application programming interface)" since we also have the gRPC API under the hood.

> Here, you can find the application programming interfaces for 
> such things as launching Fluent, assorted utilities, as well as 
> the interfaces for the meshing and solver components of Fluent.

Reviewer: ```suggestion
Here, you can find methods for launching Fluent, assorted utilities, and Fluent
meshing and solver components.
```
